src/ambulance_game/markov/markov.py

```diff
@@ -733,7 +734,7 @@ def mean_waiting_time_formula(
         if patient_type == "ambulance":
             mean_waiting_time = np.sum(
                 [
-                    max(min(state[1] + 1, threshold) - num_of_servers, 0)
+                    (min(state[1] + 1, threshold) - num_of_servers)
```

Comment:
Why this change?

Reply:
This is an earlier commit from the other branch that I cherry picked to fix the tests that where failing. It's a completely different thing.

Reply:
Ok can you remind me why the change anyway? (I'm sure it's correct just trying to remember)

Reply:
Sure. That is where I calculate the mean waiting time formula, and I used to do that by having a minimisation function inside a maximisation function but in one of our meetings you told me that I could just leave out the max( ) part and simply change the limits that I sum over. And this is what's happening here.

Reply:
Ah yes I remember. Thanks :) :+1: